geonode/upload/api/views.py

```diff
@@ -144,6 +144,21 @@ def upload(self, request, format=None):
         if not user or not user.is_authenticated:
             return Response(status=status.HTTP_401_UNAUTHORIZED)
 
+        # Custom upload steps defined by user
+        non_interactive = json.loads(
+            request.data.get("non_interactive", "false").lower()
+        )
+        if non_interactive:
```

Comment:
We want that by default the API should not be interactive,
According to @giohappy will be better to:

- change the name from non_interactive to interactive
- by default interactive will be `True`
- this line will change into `if not interactive`

Reply:
Updated.
It's aligned with the frontend?
The [upload page](https://master.demo.geonode.org/catalogue/#/upload/dataset) makes uses of this endpoint, so that change new default option can change the behaviour of the page.

Reply:
Ok.